autoPyTorch/pipeline/components/setup/network/base_network.py

```diff
@@ -106,24 +111,35 @@ def predict(self, loader: torch.utils.data.DataLoader) -> torch.Tensor:
         """
         Performs batched prediction given a loader object
         """
```

Comment:
Could you change the doc-string?

Reply:
on that note, we should document all the functions. I see there are plenty with arguments that are not documented.